<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileLoad</class>
 <widget class="QWidget" name="FileLoad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>292</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>635</width>
    <height>292</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FileLoad</string>
  </property>
  <widget class="QLabel" name="labelWind">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>站点及路线文件加载</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelStop">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>站点文件</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelLine">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>线路文件</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStop">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLine">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDone">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditStop">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditLine">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelHits">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>551</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelOrg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>地点文件</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditOrg">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOrg">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>190</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览...</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonDone</sender>
   <signal>clicked()</signal>
   <receiver>FileLoad</receiver>
   <slot>checkFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonLine</sender>
   <signal>clicked()</signal>
   <receiver>FileLoad</receiver>
   <slot>inputLine()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonQuit</sender>
   <signal>clicked()</signal>
   <receiver>FileLoad</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStop</sender>
   <signal>clicked()</signal>
   <receiver>FileLoad</receiver>
   <slot>inputStop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonOrg</sender>
   <signal>clicked()</signal>
   <receiver>FileLoad</receiver>
   <slot>inputOrg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>inputStop()</slot>
  <slot>inputLine()</slot>
  <slot>checkFile()</slot>
  <slot>inputOrg()</slot>
 </slots>
</ui>
